new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/TraefikParams.java

```diff
@@ -36,8 +49,18 @@ public HelmParams getHelmParams() {
   }
 
   public HashMap<String, Object> getValues() {
```

Comment:
Don't return concrete type like HashMap, return Map<String, Object>

Reply:
Other classes in the test framework also returns HashMap. Do you want to replace them all?

Reply:
You might as well.  The return types of methods should usually be interface types.  This makes it easier to change the concrete type later if needed.

Reply:
done